app/models.py

```diff
@@ -38,15 +38,46 @@ class Framework(db.Model):
     __tablename__ = 'frameworks'
 
     id = db.Column(db.Integer, primary_key=True)
-
     name = db.Column(db.String(255), nullable=False)
-
     expired = db.Column(db.Boolean, index=False, unique=False,
                         nullable=False)
 
 
-class Supplier(DbModelExtended):
+class User(db.Model):
+    __tablename__ = 'users'
+
+    id = db.Column(db.Integer, primary_key=True)
+    name = db.Column(db.String, index=False, unique=False,
+                     nullable=False)
+    email_address = db.Column(db.String, index=True, unique=True,
+                              nullable=False)
+    password = db.Column(db.String, index=False, unique=False,
+                         nullable=False)
+    active = db.Column(db.Boolean, index=False, unique=False,
+                       nullable=False)
+    locked = db.Column(db.Boolean, index=False, unique=False,
+                       nullable=False)
+    created_at = db.Column(db.DateTime, index=False, unique=False,
+                           nullable=False)
+    updated_at = db.Column(db.DateTime, index=False, unique=False,
+                           nullable=False)
+    password_changed_at = db.Column(db.DateTime, index=False, unique=False,
+                                    nullable=False)
+
+    def serialize(self):
+        return {
+            'id': self.id,
+            'email_address': self.email_address,
+            'name': self.name,
+            'active': self.active,
+            'locked': self.locked,
+            'created_at': self.created_at,
+            'updated_at': self.updated_at,
+            'password_changed_at': self.password_changed_at,
```

Comment:
Should `password_changed_at` be a part of the response? It feels sensitive, and I don't think there's any use for it outside the API.